src/content/patterns/creation-flows/creation-flows.mdx

```diff
@@ -0,0 +1,164 @@
+---
+title: Creation flows
+---
+
+
+<ImageComponent cols="8">
+
+![Example](images/1.png)
+
+</ImageComponent>
+
+
+#### Status:
+
+Experimental
+
+## Options
+
+<AnchorLinks>
+
+- [Standard creation](#standard-creation)
+- [Multistep creation](#multistep-creation)
+- [confirmation](#confirmation)
+
+</AnchorLinks>
+
+
+## Standard Creation
+
```

Comment:
Maybe add an intro sentence or two about what makes up or qualifies something to use a standard creation pattern. Something that makes the user understand/chose why a standard flow might be the better choice than the Multi-step flow.